### PR TITLE
Unblock old version dependencies

### LINKED ISSUE #84: rspec-retry-0.5.5 not working with latest rspec-core (3.7.0)
Failure/Error: raise Gem::LoadError, "Unable to activate #{self.full_name}, because #{y}"
Gem::LoadError:
  Unable to activate rspec-retry-0.5.5, because rspec-core-3.7.0 conflicts with rspec-core (< 3.7, > 3.3)

 /usr/share/rubygems/rubygems/specification.rb:2030:in `raise_if_conflicts'
 /usr/share/rubygems/rubygems/specification.rb:1176:in `activate'
 /usr/share/rubygems/rubygems.rb:186:in `rescue in try_activate'
 /usr/share/rubygems/rubygems.rb:183:in `try_activate'

### PR DESCRIPTION
**Pull request by avit**
Closes #84

### CODE REVIEW

**Comment by avit** [comment]
Duplicates #81

**Comment by valscion** [comment]
Seems like this issue can be closed as #81 was merged?

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,13 @@
 source 'https://rubygems.org'
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2')
+  gem "pry-debugger"
```
Comment (michaelglass) [inline Gemfile]: considering the lock files, I don't think this is working...

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ specified number of times until the example succeeds.
 
 | Rspec Version | Rspec-Retry Version |
 |---------------|---------------------|
-| 3.3-3.6       | 0.5.5               |
+| 3.3+          | 0.5.5               |
```
Comment (michaelglass) [inline README.md]: I don't like boundless compatibility assertions, e.g. 3.7 right no is YMMV

**Comment by avit** [comment]
Happy to revise this if you can still take anything from it. Let me know.

File: Gemfile (hunk 1 above)
Comment (avit) [reply Gemfile]: Not sure what you mean here, this is only the Gemfile for development and not gemspec for distribution. The lockfile shouldn't be included anywhere and gets rebuilt based on the current ruby environment for gem development.

?

**Pull request closed without merging**